src/components/Auth/Login/Login.react.js

```diff
@@ -205,9 +205,20 @@ class Login extends Component {
 		  },
 		};
 		return (
-			<div className="loginForm">
-				<Paper zDepth={0} style={styles}>
-					<h1>Login to SUSI</h1>
+			<div>
+				<div>
+                	<header className='message-thread-heading'>
+                    	<AppBar
+                        	className="app-bar"
```

Comment:
add this here
 iconElementLeft={<IconButton></IconButton>}
we do not want any hamburger icon here, other than this everything looks good.
Thanks